demos/common/include/samples/performance_metrics.hpp

```diff
@@ -36,6 +36,20 @@ class PerformanceMetrics {
                 double fontScale = 0.75,
                 cv::Scalar color = {200, 10, 10},
                 int thickness = 2);
+    void update(TimePoint lastRequestStartTime);
+
+    /// Paints metrics over provided mat
+    /// @param frame frame to paint over
+    /// @param position left top corner of text block
+    /// @param fontScale font scale
+    /// @param color font color
+    /// @param thickness font thickness
+    void paintMetrics(cv::Mat& frame,
+        cv::Point position = { 15, 30 },
+        double fontScale = 0.75,
+        cv::Scalar color = { 200, 10, 10 },
+        int thickness = 2) const;
+
```

Comment:
They aren't needed now.

Reply:
Ping

Reply:
they are needed as paintMetrics might be called by application